modules/cnv_vm_events_web.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-vm-events-web]]
+== Viewing the events of a virtual machine
+
+You can view the events for a running a virtual machine from the *Virtual Machine Details* screen of the web console.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines*  from the side menu.
+. Select a Virtual Machine.
+. Click *Events* to view all events for the virtual machine. 
+
```

Comment:
Maybe add a note about Streaming Events and Pausing?

Reply:
Added lines about pausing and streaming before the procedure.